app/jobs/fedora3_ingestor.rb

```diff
@@ -27,7 +27,7 @@ def add_datastreams_to_page(page, image_data)
   end
 
   def add_datastreams_to_document(document, document_data)
-    document_data.add_datastream_from_file "#{@job.dir}/#{document.filename}", 'MASTER', 'application/pdf'
+    document_data.add_datastream_from_file "#{@job.dir}/#{document}", 'MASTER', 'application/pdf'
```

Comment:
I don't think you want to change this file. This is used by the normal ingest process; I think you copied it and changed it in a different file, so this version should be left alone.